lib/v2/hk01/channel.js

```diff
@@ -0,0 +1,40 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+const { parseDate } = require('@/utils/parse-date');
+
+module.exports = async (ctx) => {
+    const id = ctx.params.id;
+
+    const response = await got(`https://web-data.api.hk01.com/v2/page/category/${id}`);
+    const data = response.data;
+    const list = data.sections[0].items;
+
+    await Promise.all(
+        list.map((item) =>
+            ctx.cache.tryGet(item.data.canonicalUrl, async () => {
```

Comment:
```suggestion
            ctx.cache.tryGet(`https://www.hk01.com/${item.data.mainCategory}/${item.data.articleId}`, async () => {
```
or
```js
item.data.canonicalUrl.substring(0, item.data.canonicalUrl.lastIndexOf('/'));
```
to avoid cache invalidation in case the encoded title in the url is updated.